acr.patch

```diff
@@ -0,0 +1,32 @@
+diff --git a/build/build-config-edit.sh b/build/build-config-edit.sh
```

Comment:
Was this added by mistake?

Reply:
This patch is required as https://github.com/Azure/acr-docker-credential-helper is hard coded to build for amd64.